syftlib/src/main/java/org/openmined/syft/networking/requests/HttpAPI.kt

```diff
@@ -25,6 +25,8 @@ import retrofit2.http.Streaming
 
 interface HttpAPI : CommunicationAPI {
 
+    // HttpAPI is a interface which helps in calling correct federated related http endpoints.
```

Comment:
This should be above the interface initiation and better to put it in a Kdoc instead of inline. Also mention it implements some of communication API which are available for both http and socket endpoint